sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -1098,33 +412,57 @@ public virtual SearchAddressBatchOperation SearchAddressBatch(WaitUntil waitUnti
         }
 
         /// <summary>
-        /// The Search Address Reverse Batch API sends batches of queries to <see href="https://docs.microsoft.com/rest/api/maps/search/getsearchaddressreverse">Search Address Reverse API</see> using just a single API call. This Search Address Reverse Batch API can handle up to **10000** queries.
+        ///
+        /// The Reverse Geocoding Batch API sends batches of queries to [Reverse Geocoding API](/rest/api/maps/search/get-reverse-geocoding) using just a single API call. The API allows caller to batch up to <c>100</c> queries.
+        ///
+        /// ### Submit Synchronous Batch Request
```

Comment:
Suggest to remove "### Submit Synchronous Batch Request", "### POST Body for Batch Request", and "### Batch Response Model" sections

Reply:
@dubiety done